### ISSUE TITLE
Replicate Model 2 from cdcgov/wastewater-informed-covid-forecasting

### ISSUE THREAD

**Issue by ghost** [CDCgov/multisignal-epi-inference#3]
## Goal

Have a working Python library that implements the most basic version of the wastewater model. By most basic, we mean (a) hospitalizations only, and (b) single geographical unit (no pooled model).

## Context

See #32.

## Required features

- The model (implemented either as a function or class) should be able to read hospitalization data simulated from the WW model.
- It should fit the model using the no-U turn sampler.
- Return the estimates.
- And make predictions.

## Specifications

- [x] Create (or bring in from WW) data for testing.
- [x] Write a Python script under `model/src` implementing the model (like [this](https://github.com/cdcent/cfa-multisignal-renewal/blob/236172bf93e470da3be515a8fbb3e6d69aa735f0/model/src/pyrenew/basic.py))
- [x] Document the model.
- [x] Write a Jupyter notebook walking through the process of read/fit/predict (vignette).
- [ ] Have a test that compares the estimates of this Python module with the WW R package.

## Out of scope

Features beyond the basic model.

## Related documents

- See #18 for math equations.
- See the Stan implementation in the WW R package [here](https://github.com/cdcent/cfa-forecast-renewal-ww/blob/aac9ac333b9e7f0deeac5974ec553d13f713d440/cfaforecastrenewalww/inst/stan/renewal_ww_hosp.stan)

Ref: https://github.com/cdcent/cfa-multisignal-renewal/issues/44
author: @gvegayoncdc

**Comment by gvegayoncdc** [comment]
@afmagee42 and @kaitejohnson, could you point me to the data + estimates you use for CI in the WW model 👼?

**Comment by afmagee42** [comment]
For our CI on the model, our testing inputs and expected outputs are stored in an [`.rda` file](https://github.com/cdcent/cfa-forecast-renewal-ww/blob/main/cfaforecastrenewalww/R/sysdata.rda). These are produced in [`internal_data.R`](https://github.com/cdcent/cfa-forecast-renewal-ww/blob/main/cfaforecastrenewalww/data-raw/internal_data.R), using package-public data. The internal testing data are stan data objects and single stan output iterations from short runs.

The public example data is generated in [`example_data.R`](https://github.com/cdcent/cfa-forecast-renewal-ww/blob/main/cfaforecastrenewalww/data-raw/example_data.R) and live in the package's [`data` folder](https://github.com/cdcent/cfa-forecast-renewal-ww/tree/main/cfaforecastrenewalww/data). These are simulations of semi-processed data (post-pull, pre stan-preprocessing).

(Yes this is a bit messy, blame me and my insistence that users should not have access to purely testing data in the package, because that's why it's this way.)

Pipeline testing/CI is handled at the repo level and currently just checks that the pipeline runs without errors.

**Comment by afmagee42** [comment]
We specifically don't use any real data or estimates in order to make sure there's no potential data leak, as this lives in the public repo.

Our first-pass at CI data for checking the stan output was consistent was taken from a posterior-predictive dataset from a real model run.

The testing would have also included (or perhaps just been) the computation of the joint posterior density at some arbitrary parameter values, but cmdstanr won't expose the requisite functions in WSL so we had to evaluate based on MCMC output instead. (I don't like that this compounds both changes in the model and any changes stan makes to the algorithm in one place, but it's what we've got.)
